research/src/pages/popup/popup.research.explainer.mdx

```diff
@@ -155,27 +203,138 @@ function supportsPopup() {
+
+```html
+<div popup=popup>I'm a popup</div>
+<style>
+[popup]::backdrop {
+  backdrop-filter: blur(3px);
+}
+</style>
+```
+
+Note that in contrast to the `::backdrop` pseudo element for modal dialogs and fullscreen elements, the `::backdrop` for a popup is styled by a UA stylesheet rule `pointer-events: none !important`, which means it cannot trap clicks outside the popup. This ensures the "click outside" light dismiss behavior continues to function.
+
+# Behaviors
+
+
+
+## Automatic Dismiss Behavior
+
+For elements that are displayed on the top layer via this API, there are a number of things that can cause the element to be removed from the top-layer, besides the ones [described above](#showing-and-hiding-a-popup). These fall into three main categories:
+
+* **One at a Time.** Another element being added to the top-layer causes existing top layer elements to be removed. This is typically used for “one at a time” type elements: when one popup is shown, other popups should be hidden, so that only one is on-screen at a time. This is also used when “more important” top layer elements get added. For example, fullscreen elements should close all open popups.
+* **Light Dismiss**. User action such as clicking outside the element, hitting Escape, or causing keyboard focus to leave the element should all cause a showing popup to be hidden. This is typically called “light dismiss”, and is discussed in more detail in [this section](#light-dismiss).
 * **Other Reasons**. Because the top layer is a UA-managed resource, it may have other reasons (for example a user preference) to forcibly remove elements from the top layer.
 
-In all such cases, the UA manages the removal of elements from the top layer, by forcibly removing the element from the top layer, and re-applying the `display:none` UA rule.
+In all such cases, the UA is allowed to forcibly remove an element from the top layer and re-apply the `display:none` popup UA rule. The rules the UA uses to manage these interactions depends on the element types, and this is described in more detail in [this section](#classes-of-top-layer-ui).
+
+### Light Dismiss
+
+The term "light dismiss" for a popup is used to describe the user "moving on" to another part of the page, or generally being done interacting with the popup. When this happens, the popup should be hidden. Several actions trigger light dismiss of a popup:
+
+1. Clicking or tapping outside the bounds of the popup. Any `mousedown` event will trigger **all open popups** to be hidden, starting from the top of [the popup stack](#the-popup-stack), and ending with the [nearest open ancestral popup](#nearest-open-ancestral-popup) of the `mousedown` event's `target` `Node`. This means clicking on a popup or its trigger or anchor elements will not hide that popup.
```

Comment:
suggestion, maybe `mouseup` would be good here?  

I say this because [WCAG 2.5.2 Pointer Cancellation criterion](https://www.w3.org/WAI/WCAG21/Understanding/pointer-cancellation.html) could then be adhered to, allowing someone who makes an accidental down click outside of a popup (someone with motor disabilities, for instance) would have the chance to move their pointer back into the boundaries of the popup, allowing for the accidental dismissal to be aborted.

Reply:
This is likely worth discussion. I did a rather informal survey of native and web popups, and found that nearly all of them dismiss on mouse-down. Think it's worth an issue?

Reply:
i think it's definitely worth at least a discussion.

there are presently a lot of platform behaviors that get a 'pass' with wcag rules because web authors can literally do nothing about them, since they're baked into the platform.  but, if given the chance to change a behavior, it at least deserves being brought up and then further discussion can dictate if there's a reason to adhere to what's been previously done vs going in a different direction.  

thanks!

Reply:
Filed! https://github.com/openui/open-ui/issues/529